Resources/public/js/components/price-list/main.js

```diff
@@ -122,10 +130,23 @@ define([], function() {
             var bulkPriceComponents = [];
 
             this.sandbox.util.foreach(this.options.currencies, function(currency){
+
+                this.bulkPriceData = [];
```

Comment:
and please make sure u fix the other occurrences of this problem too.. 

 like here https://github.com/sulu-io/SuluProductBundle/blob/feature/special-price-frontend/Resources/public/js/components/price-list/main.js#L93